cms/djangoapps/contentstore/signals/handlers.py

```diff
@@ -177,8 +177,9 @@ def listen_for_xblock_published(sender, signal, **kwargs):
     Publish XBLOCK_PUBLISHED signals onto the event bus.
     """
     if settings.FEATURES.get("ENABLE_SEND_XBLOCK_EVENTS_OVER_BUS"):
+        topic = getattr(settings, "EVENT_BUS_XBLOCK_LIFECYLCE_TOPIC", "course-authoring-xblock-lifecycle")
```

Comment:
shouldn't all these topics be named `EVENT_BUS_TOPIC_PREFIX+EVENT_BUS_XBLOCK_LIFECYLCE_TOPIC `

Reply:
As mentioned above, it is combined by underlying implementation.